book/src/concepts/event-channel.md

```diff
@@ -0,0 +1,120 @@
+# Event Channel
+
+This chapter will be easier than the previous ones. 
+
+While it is not essential to understand it to use amethyst, it can make your life much much easier in a lot of situations where using only data would make your code too complex.
+
+## What is an event channel?
+
+An `EventChannel` acts like a queue for any type that implements `Event`.
+The `Event` type is automatically implemented for all types that are `Send`, `Sync` and have a static lifetime.
+
+It is a single producer/multiple receiver. This means that it should be used when you only have a single "thing" (usually a system) producing events.
```

Comment:
Not sure I agree with this advice.  Multiple producers of events is fine, but it should be understood that they can't all write events in simultaneously.  Multiple reading is definitely simultaneous though.